api/api.go

```diff
@@ -201,6 +201,7 @@ type Rates []Rate
 type Tariff interface {
 	Unit() string
 	Rates() (Rates, error)
+	Dynamic() bool
```

Comment:
Find ich da unglücklich. Dann lieber als `tariff.IsDynamic(t *Tariff) bool` package function die sich einfach den Plan anschaut und guckt ob alle Slots identisch sind?

Reply:
Ich brauche die Information um zu entscheiden ob wir den Dialog für das Limit und die Visualisierung überhaupt in der UI anbieten. Daher würd ich ungern vorab schon die Tarifdaten laden nur um das dann in der UI rauszufinden.

Rename ist drin.

Reply:
> Daher würd ich ungern vorab schon die Tarifdaten laden nur um das dann in der UI rauszufinden.

Die Daten werden ohnehin geladen, spielt also keine Rolle.

Reply:
Ist aber auch egal...